### DISCUSSION TITLE
Update lookup field Set fixed value always blank

### DISCUSSION THREAD

**Discussion by rodmarialvas** [Innofactor/XrmToolBox.BulkDataUpdater#221] [Q&A] [answered]
Hi everyone,

I've been trying to update some records from one lookup value to another one but when I select from the list it just goes blank as shown below.

Anyone faced this before?

![image](https://github.com/rappen/BulkDataUpdater/assets/70517004/aeb73012-7d20-41fa-a30b-7b08a363111b)

![image](https://github.com/rappen/BulkDataUpdater/assets/70517004/42e543fb-c574-450b-bf33-e45e3c114e70)

Thanks
Rodolfo

**Comment by rappen** [comment]
Aouch. That seems to be an ugly bug.
There will probably be big-fix-release pretty soon.

That's for the report!

**Comment by rappen** [comment, marked as answer]
This bug is now fixed - update BDU to version 1.2024.6.x or later!

**Reply by rodmarialvas** [reply]
Just tested and working great. Thanks again.
